### PR TITLE
Add new words to zombie, mouse, and cat accent

### PR DESCRIPTION
**Pull request by RIKELOLDABOSS**
## About the PR
i added some more words for the zombies and mice to say.

## Why / Balance
because its cool! and also, i wanted to do something simple for my first project.

## Technical details

its just some more words for the mice and zombies

## Media
image
image

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
--> 

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Resources/Locale/en-US/accent/accents.ftl [hunk 1]
```diff
@@ -4,6 +4,8 @@ accent-words-cat-2 = Mow.
 accent-words-cat-3 = Mrrrow!
 accent-words-cat-4 = Hhsss!
 accent-words-cat-5 = Brrow.
+accent-words-cat-6 = meow?
+accent-words-cat-7 = miau
```
Comment (Visne) [inline Resources/Locale/en-US/accent/accents.ftl:8]: Should be capitalized

File: Resources/Prototypes/Accents/full_replacements.yml [hunk 2]
```diff
@@ -59,6 +62,9 @@
   - accent-words-zombie-5
   - accent-words-zombie-6
   - accent-words-zombie-7
+  - accent-words-zombie-8
+  - accent-words-zombie-9
+  - accent-wors-zombie-10
```
Comment (Visne) [inline Resources/Prototypes/Accents/full_replacements.yml:67]: Missing `accent-words-cat-6` and `accent-words-cat-7` in this file

File: CppProperties.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      "defines": [
+        "WIN32",
+        "_DEBUG",
+        "UNICODE",
+        "_UNICODE"
+      ],
+      "intelliSenseMode": "windows-msvc-x86"
+    }
+  ]
+}
```
Comment (Visne) [inline CppProperties.json:21]: Need to get rid of this file

**Review by Visne — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Accents/full_replacements.yml (hunk 2 above)
Comment (RIKELOLDABOSS) [reply Resources/Prototypes/Accents/full_replacements.yml:67]: damn, i forgot about adding more cat stuff

File: CppProperties.json (hunk 3 above)
Comment (RIKELOLDABOSS) [reply CppProperties.json:21]: yeah, i dont know what it is

File: Resources/Prototypes/Accents/full_replacements.yml (hunk 2 above)
Comment (RIKELOLDABOSS) [reply Resources/Prototypes/Accents/full_replacements.yml:69]: all done

**Review by RIKELOLDABOSS — COMMENTED** [review]
all done

**Review by Visne — APPROVED** [review]

**Pull request merged**